### ISSUE TITLE
feature request: add option to display caller memory address in drstrace

### ISSUE THREAD

**Issue by psec** [DynamoRIO/drmemory#2056]
It would be useful to have the option to display the address of the caller or the return address in drstrace. 

For example, the output under this option would read:

NtOpenSection
        ret addr: 0x00405340
	arg 0: 0x001ffb5c (type=HANDLE*, size=0x4)
	arg 1: 0xf (type=unsigned int, size=0x4)
        ...

**Comment by derekbruening** [comment]
@mxmssh already added this I believe?

**Comment by mxmssh** [comment]
we have this option in drltrace (-print_ret_addr).

**Comment by derekbruening** [comment]
Right.  For drstrace I would think just the direct caller would not normally be sufficient: it's going to always be a winapi routine (your NtOpenSection example must be some unusual app doing raw syscalls?).  A callstack to connect it back to the app would be more useful.

**Comment by derekbruening** [comment]
A callstack feature is already under #1756.  I would say close this one in favor of that.
